src/datasets/utils/tf_utils.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 The HuggingFace Datasets Authors and the TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""TF-specific utils import."""
+
+import numpy as np
+import pyarrow as pa
+
+from .. import config
+
+
+def minimal_tf_collate_fn(features):
+    if config.TF_AVAILABLE:
+        import tensorflow as tf
+    else:
+        raise ImportError("Called a Tensorflow-specific function but Tensorflow is not installed.")
+
+    first = features[0]
+    batch = {}
+    for k, v in first.items():
+        if isinstance(v, np.ndarray):
+            batch[k] = np.stack([f[k] for f in features])
+        elif isinstance(v, tf.Tensor):
+            batch[k] = np.stack([f[k].numpy() for f in features])
```

Comment:
```suggestion
            batch[k] = tf.stack([f[k] for f in features])
```
(Or is there a particular reason to use numpy's stack? If the goal is to have a numpy object, we can call `.numpy()` after stacking, which should be faster :D)

<img width="1088" alt="Screenshot 2022-04-27 at 17 06 28" src="https://user-images.githubusercontent.com/12240844/165562654-dd8422be-5836-4fb9-bdb8-50ce28622ff2.png">

Reply:
You're right, this was an oversight. Thanks!